http://sourceware.org/ml/libc-alpha/2011-06/msg00087.html

binutils started emitting ELF files with ELFOSABI_GNU due to GNU-specific
ELF extensions. We need to accept loading such binaries on hurd-i386 too.

2011-06-19  Samuel Thibault  <samuel.thibault@ens-lyon.org>

        * elf/elf.h (ELFOSABI_GNU): Define macro.
        (ELFOSABI_LINUX): Comment that it is just an alias for ELFOSABI_GNU.

---
 elf/elf.h |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/elf/elf.h
+++ b/elf/elf.h
@@ -142,7 +142,8 @@
 #define ELFOSABI_SYSV		0	/* Alias.  */
 #define ELFOSABI_HPUX		1	/* HP-UX */
 #define ELFOSABI_NETBSD		2	/* NetBSD.  */
-#define ELFOSABI_LINUX		3	/* Linux.  */
+#define ELFOSABI_GNU		3	/* GNU.  */
+#define ELFOSABI_LINUX		3	/* Alias for ELFOSABI_GNU.  */
 #define ELFOSABI_SOLARIS	6	/* Sun Solaris.  */
 #define ELFOSABI_AIX		7	/* IBM AIX.  */
 #define ELFOSABI_IRIX		8	/* SGI Irix.  */
